lib/widgets/user_profile.dart

```diff
@@ -215,117 +215,95 @@ class _UserOverview extends HookWidget {
-                          Icon(
-                            Icons.article,
-                            size: 15,
-                            color: colorOnTopOfAccentColor,
-                          ),
-                          Padding(
-                            padding: const EdgeInsets.only(left: 4),
-                            child: Text(
-                              '${compactNumber(userView.counts.postCount)}'
-                              ' Post${pluralS(userView.counts.postCount)}',
-                              style: TextStyle(color: colorOnTopOfAccentColor),
-                            ),
-                          ),
-                        ],
-                      ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Chip(
+                    label: Row(
+                      children: [
+                        Icon(
+                          Icons.article,
+                          size: 15,
+                          color: colorOnTopOfAccentColor,
+                        ),
+                        const SizedBox(width: 4),
+                        Text(
+                          '${compactNumber(userView.counts.postCount)}'
+                          ' Post${pluralS(userView.counts.postCount)}',
+                          style: TextStyle(color: colorOnTopOfAccentColor),
+                        ),
+                      ],
                     ),
-                    Padding(
-                      padding: const EdgeInsets.only(left: 16),
-                      child: Badge(
-                        child: Row(
-                          children: [
-                            Icon(
-                              Icons.comment,
-                              size: 15,
-                              color: colorOnTopOfAccentColor,
-                            ),
-                            Padding(
-                              padding: const EdgeInsets.only(left: 4),
-                              child: Text(
-                                '${compactNumber(userView.counts.commentCount)}'
-                                ''' Comment${pluralS(userView.counts.commentCount)}''',
-                                style:
-                                    TextStyle(color: colorOnTopOfAccentColor),
-                              ),
-                            ),
-                          ],
+                  ),
+                  const SizedBox(width: 16),
+                  Chip(
+                    label: Row(
+                      children: [
+                        Icon(
+                          Icons.comment,
+                          size: 15,
+                          color: colorOnTopOfAccentColor,
                         ),
-                      ),
+                        const SizedBox(width: 4),
+                        Text(
+                          '${compactNumber(userView.counts.commentCount)}'
```

Comment:
is there a reason why this is inside of a string?
why `'${compactNumber(userView.counts.commentCount)}'` instead of `compactNumber(userView.counts.commentCount)`?

Reply:
Yes, it is concatenated with the string below it. I'll merge them